num_of_buttons: 15 
axis_linear_x: 1
axis_linear_y: 0
axis_angular: 2
axis_linear_z: 3
scale_linear: 1
scale_angular: 1
scale_linear_z: 20.0 
button_speed_up: 3
button_speed_down: 1
button_dead_man: 5
button_output_1: 15
button_output_2: 13
output_1: 1 
output_2: 2
button_ptz_tilt_up: 4
button_ptz_tilt_down: 6
button_ptz_pan_right: 5
button_ptz_pan_left: 7
button_ptz_zoom_wide: 3
button_ptz_zoom_tele: 0
button_home: 8
tilt_increment: 5
pan_increment: 5
zoom_increment: 300
button_kinematic_mode: 7
cmd_service_set_mode: /vulcano_base_controller/set_mode
cmd_service_home:  /vulcano_base_controller/home
joystick_dead_zone: "true" 
delta_angle: 0.15
